docs/index.md

```diff
@@ -5,19 +5,55 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-## Getting started
+.NET Documentation
+==================
 
-In this docset, you will find tutorials that will teach you how to get started with .NET Core. You can also find the API reference documentation. 
+Welcome to .NET! You can build any kind of app you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next app today, for free, on Windows, OS X and Linux.
```

Comment:
@tdykstra That's an excellent point. I was struggling with that issue. There are two things at play here that we need to rationalize:

- This is the new home for .NET docs. Most of the docs are not here yet. We position the whole scope of .NET from the start or can more content to this page as the doc-sets arrive. I went with the first option.
- The flipside is that it is not a great user experience when you end up somewhere else.

Thoughts?